src/components/modules/tools.ts

```diff
@@ -1,19 +1,21 @@
-import Paragraph from '../tools/paragraph/dist/bundle';
+import Paragraph from '../../tools/paragraph/dist/bundle';
 import Module from '../__module';
 import * as _ from '../utils';
 import {
-  BlockToolConstructable,
   EditorConfig,
-  InlineTool,
-  InlineToolConstructable, Tool,
-  ToolConfig,
+  Tool,
   ToolConstructable,
   ToolSettings
 } from '../../../types';
 import BoldInlineTool from '../inline-tools/inline-tool-bold';
 import ItalicInlineTool from '../inline-tools/inline-tool-italic';
 import LinkInlineTool from '../inline-tools/inline-tool-link';
-import Stub from '../tools/stub';
+import Stub from '../../tools/stub';
```

Comment:
The Stub tool is just an implementation of the Block Tool, so it should not be placed at the same level as abstract tools classes. It would better to move it to the `tools/block/stub`

Reply:
There is a confusion about dir names. We have tools for paragraph and stub, I just moved it one level up